src/tests/pws.rs

```diff
@@ -130,8 +146,11 @@ fn set_reset_get(model: nitrokey::Model) -> anyhow::Result<()> {
 
 #[test_device]
 fn clear(model: nitrokey::Model) -> anyhow::Result<()> {
+  clear_pws(model)?;
+
   let mut ncli = Nitrocli::new().model(model);
-  let _ = ncli.handle(&["pws", "set", "10", "clear-test", "some-login", "abcdef"])?;
+  let _ = ncli.handle(&["pws", "clear", "10"])?;
```

Comment:
Hm, why is that call needed?

Reply:
It’s not needed, but I figured it would be good to check that we can clear an unprogrammed slot without errors.  But we can also move that into a separate test case.